.pre-commit-config.yaml

```diff
@@ -36,3 +36,47 @@ repos:
           - types-aiobotocore[essential]
           - boto3-stubs[essential]
         exclude: ^(diracx-client/src/diracx/client/_generated|diracx-[a-z]+/tests/|diracx-testing/|build|extensions/gubbins/gubbins-client/src/gubbins/client/_generated)
+
+-   repo: https://github.com/codespell-project/codespell
```

Comment:
https://github.com/marketplace/actions/typos-action seems to be receiving more updates than codespell, so I would prefer that.

Reply:
Well, I've just managed to ignore (with --skip argument) autogenerated code from spellchecking. The problem is that each of these tools seem to have a different way of ignoring directories. My next target is `pygrep`.... Also `blanket-noqa` complains a lot.